seismicpro/src/seismic_batch.py

```diff
@@ -160,17 +161,33 @@ def __init__(self, index, *args, preloaded=None, **kwargs):
         if preloaded is None:
             self.meta = dict()
 
-    def _init_component(self, *args, dst, **kwargs):
+    def _init_component(self, src, dst, *args, **kwargs):
```

Comment:
```suggestion
    def _init_component(self, *args, src, dst, **kwargs):
```
This way `src` and `dst` forced to be keyword

Reply:
> It seems that proposed changes "as is" will break `SeismicBatch.load`.
> 
> It calls `_load_segy(src=components, dst=components, ...)`, which calls `_load_from_segy_file`, wrapped with `inbatch_parallel(init="_init_component", ...)`. This way `_init_component` will get yet nonexistent component as `src`.

function `_load_from_segy_file` calls with `(src=components, dst=components, tslice=tslice)` as an arguments. So in this case it will works properly as well. As far as i understand we always have `src` component when use  `@inbatch_parallel` decorator.